### PR TITLE
Update BNFabb114.xml

### PR DESCRIPTION
**Pull request by GueshSol**

### CODE REVIEW

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
Two things: 1) miniatures are usually are described under "Decoration", according to my knowledge. 2) Are all miracles identified? I see that the description stops at fol. 18r, while the ms encompas many more leaves.  "Pricking and ruling" needs a formula. "Binding" also should be described.

File: ParisBNF/abb/BNFabb114.xml [hunk 1]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </msItem>
                      </msItem>
                      
+                     <msItem xml:id="ms_i1.5">
+                        <locus from="11va" to="14r"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:124]: `  <locus from="12va" to="14r"/>`

File: ParisBNF/abb/BNFabb114.xml [hunk 2]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <textLang mainLang="gez"/>
+                        <title type="complete" ref="LIT5673AkkonuColl#AkkColl3"/>
+                        
+                        <msItem xml:id="ms_i1.5.1">
+                           <locus from="11va" to="13rb"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:129]: <locus from="12va" to="13rb"/>

File: ParisBNF/abb/BNFabb114.xml [hunk 3]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i1.5.2">
+                           <locus target="#13rb"/>
+                           <title type="complete" ref="LIT4274AkkonuB#Stanza3"/>
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.3">
+                           <locus target="'#14r"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:140]: not the erroneous ' before #
`   <locus target="#14r"/>`

File: ParisBNF/abb/BNFabb114.xml [hunk 4]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        </msItem>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.6">
+                        <locus from="14va" to="15v"/>
+                        <textLang mainLang="gez"/>
+                        <title type="complete" ref="LIT5673AkkonuColl#AkkColl4"/>
+                        
+                        <msItem xml:id="ms_i1.6.1">
+                           <locus from="11va" to="13rb"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:152]: `    <locus from="14va" to="15ra"/> `

File: ParisBNF/abb/BNFabb114.xml [hunk 5]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i1.6.2">
+                           <locus from="15ra" to="15rb"/>
+                           <title type="complete" ref="LIT4274AkkonuB#Stanza4"/>
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i1.6.3">
+                           <locus target="'#15v"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:163]: `   <locus target="#15v"/>`

File: ParisBNF/abb/BNFabb114.xml [hunk 6]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <title type="complete" ref="LIT5673AkkonuColl#AkkMin4"/>
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.7">
+                        <locus from="14va" to="15v"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:170]: ` <locus from="16ra" to="18r"/>`

File: ParisBNF/abb/BNFabb114.xml [hunk 7]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <textLang mainLang="gez"/>
+                        <title type="complete" ref="LIT5673AkkonuColl#AkkColl5"/>
+                        
+                        <msItem xml:id="ms_i1.7.1">
+                           <locus from="16ra" to="18r"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:175]: `<locus from="16ra" to="16vb"/> `

File: ParisBNF/abb/BNFabb114.xml [hunk 8]
```diff
@@ -120,6 +120,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <title type="complete" ref="LIT5673AkkonuColl#AkkColl5"/>
+                        
+                        <msItem xml:id="ms_i1.7.1">
+                           <locus from="16ra" to="18r"/> 
+                           <title type="complete" ref="LIT3591Miracle"/>
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i1.7.2">
+                           <locus target="'#16vb"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb114.xml:181]: `  <locus target="#16vb"/>`

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, Guesh! This is already excellent work. All items are identified correctly and the msItems are all nested correctly. There are only a few small mistakes regarding the folio numbers, due to copy pasting. Can you correct them on the same branch and then commit again?

**Comment by thea-m** [comment]
@DenisNosnitsin1970 : You are right, the description is not complete. We are doing it step by step to practice the PR workflow.

**Review by thea-m — APPROVED** [review]
Very good!

**Pull request merged**